pkg/asset/manifests/cloudproviderconfig.go

```diff
@@ -30,7 +32,10 @@ import (
 )
 
 var (
-	cloudProviderConfigFileName = filepath.Join(manifestDir, "cloud-provider-config.yaml")
+	cloudProviderConfigFileName         = filepath.Join(manifestDir, "cloud-provider-config.yaml")
+	aroCloudProviderRoleFileName        = filepath.Join(manifestDir, "aro-cloud-provider-secret-reader-role.yaml")
+	aroCloudProviderRoleBindingFileName = filepath.Join(manifestDir, "aro-cloud-provider-secret-reader-rolebinding.yaml")
+	aroCloudProviderSecretFileName      = filepath.Join(manifestDir, "aro-cloud-provider-secret.yaml")
```

Comment:
Cluster secrets and the secret readers are already created here: https://github.com/openshift/installer/blob/master/pkg/asset/manifests/openshift.go
with the templates here:
https://github.com/openshift/installer/tree/master/data/data/manifests/openshift

Wouldn't it make more sense to move this logic there?

Reply:
Apologies, but my first link was incorrect. The logic for generating those templates is here: https://github.com/openshift/installer/tree/master/pkg/asset/templates/content/openshift
 
I do think we should use this pattern of generating the resources from a template, but I wonder if it makes more sense to keep that logic in the cloud provider config as you originally had it or to move it to `templates`. @staebler what do you think?